src/components/list/list-base.scss

```diff
@@ -0,0 +1,300 @@
+/** External component calculations */
+$md-checkbox-width: 3 * $baseline-grid !default;
+//TODO(devversion): use shared variable from buttons.scss
```

Comment:
If it's for future work please file an issue and write it's number on the comment